src/components/map/map.tsx

```diff
@@ -268,6 +380,16 @@ export class Map implements GxComponent {
             coords={this.userLocationCoords}
           ></gx-map-marker>
         )}
+        {this.selectionLayer && this.isSelectionLayerSlot ? (
+          <slot name="selection-layer-marker" />
+        ) : (
+          <gx-map-marker
+            marker-class="gx-default-selection-layer-icon"
+            icon-width="30"
+            icon-height="30"
+            coords={this.centerCoords}
+          ></gx-map-marker>
+        )}
```

Comment:
This condition is what makes the render method introduce the default maker when `selectionLayer` is false

```suggestion
        this.selectionLayer && {this.isSelectionLayerSlot ? (
          <slot name="selection-layer-marker" />
        ) : (
          <gx-map-marker
            marker-class="gx-default-selection-layer-icon"
            icon-width="30"
            icon-height="30"
            coords={this.centerCoords}
          ></gx-map-marker>
        )}
```

Reply:
Interesting suggestion 🤔 
I will apply those changes too! But I found the bug in the handler of  `MapMarkerDidLoad` event. I will fix that and apply your suggestion!

Reply:
It does not works. When you move the `{` the code behind.

Reply:
Sorry, proposed the change but didn't properly verify it.

Let's try again:

```
        {this.selectionLayer && (this.isSelectionLayerSlot ? (
          <slot name="selection-layer-marker" />
        ) : (
          <gx-map-marker
            marker-class="gx-default-selection-layer-icon"
            icon-width="30"
            icon-height="30"
            coords={this.centerCoords}
          ></gx-map-marker>
        ))}
```

Reply:
This works! I made another solution, but I like yours.
My solution was:
```typescript
        let selectionMarkerElement;
	if(this.selectionLayer){
		selectionMarkerElement = this.isSelectionLayerSlot ? (
			<slot name="selection-layer-marker" />
		  ) : (
			<mkm-map-marker
			  marker-class="mkm-default-selection-layer-icon"
			  icon-width="30"
			  icon-height="30"
			  coords={this.centerCoords}
			></mkm-map-marker>
		  )
	}
```
and in the return:
```typescript
        {this.selectionLayer && selectionMarkerElement}
```

Reply:
fixed in 5262096